tests.py

```diff
@@ -1,9 +1,8 @@
 import logging
-import sys
-import unittest
 import usabilla as ub
 
 from mock import Mock
+import requests
```

Comment:
I'd like to see the import statements being ordered alphabetically. Next to that it would be great to group the `import ...` statements and then do the `from ... import ...` statements.

```python
import logging
import requests
import usabilla as ub

from mock import Mock
from unittest import TestCase, main as unittest_main
```

Reply:
👍